build broken due to incompatible build versions

Error: Multiple versions of pnpm specified:
  - version latest in the GitHub Action config with the key "version"
  - version pnpm@9.10.0+sha256.355a8ab8dbb6ad41befbef39bc4fd6b5df85e12761d2724bd01f13e878de4b13 in the package.json with the key "packageManager"
Remove one of these versions to avoid version mismatch errors like ERR_PNPM_BAD_PM_VERSION

--- a/package.json
+++ b/package.json
@@ -31,5 +31,4 @@
     "tailwindcss": "^3.4.7",
     "typescript": "^5.5.4"
   },
-  "packageManager": "pnpm@9.10.0+sha256.355a8ab8dbb6ad41befbef39bc4fd6b5df85e12761d2724bd01f13e878de4b13"
 }

